POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Services/NotificationService.cs

```diff
@@ -44,38 +131,35 @@ public static void AppInstalledToast()
                     }
                 }
             };
+            return visual;
+        }
 
+        private static ToastActionsCustom createToastAction(string title, string query)
+        {
             ToastActionsCustom actions = new ToastActionsCustom()
             {
-                Inputs = { },
                 Buttons =
-                {
-                    new ToastButton("Open PowerToys", new QueryString()
-                    {
-                        // Query for button
-                    }.ToString())
                     {
-                        // Insert activation type here
-                    },
-                }
+                        new ToastButtonDismiss("Not now"),
+                        new ToastButton("Get Started", new QueryString()
```

Comment:
Could be a future consideration: define string constants in separate file

Reply:
Will consider for future